### PR TITLE
Announcing Scala.js 0.6.10

### PR DESCRIPTION
**Pull request by sjrd**
Still have to do the list of bug fixes.

### CODE REVIEW

**Comment by sjrd** [comment]
Updated. Review by @gzm0 and @nicolasstucki

File: _posts/news/2016-06-17-announcing-scalajs-0.6.10.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Bug fixes
+
+Among others, the following bugs have been fixed:
+
+* [#2314](https://github.com/scala-js/scala-js/issues/2314) Infinite loop in `BigDecimal.isValidLong` (again)
+* [#2376](https://github.com/scala-js/scala-js/issues/2376) PhantomJSEnv does not properly escape JS code in webpage (duplicate [#2279](https://github.com/scala-js/scala-js/issues/2279), [#2322](https://github.com/scala-js/scala-js/issues/2322))
+* [#2368](https://github.com/scala-js/scala-js/issues/2368) RhinoJSEnv does not support setTimeout without second argument
+* [#2333](https://github.com/scala-js/scala-js/issues/2333) The JUnit runner does not report failures as events, causing `sbt test` to erroneously succeed
+* [#2392](https://github.com/scala-js/scala-js/issues/2392) java.util.Date should have method getTimezoneOffset, now has getTimeZoneOffset
```
Comment (gzm0) [inline _posts/news/2016-06-17-announcing-scalajs-0.6.10.md]: Consider dropping the "now has..." part. It is pretty confusing when read here.

**Comment by gzm0** [comment]
That's all.

File: _posts/news/2016-06-17-announcing-scalajs-0.6.10.md (hunk 1 above)
Comment (sjrd) [inline _posts/news/2016-06-17-announcing-scalajs-0.6.10.md]: OK

**Comment by nicolasstucki** [comment]
LGTM

**Comment by sjrd** [comment]
Updated. I will merge tomorrow morning.

**Pull request merged**